docs/src/main/sphinx/release/release-462.md

```diff
@@ -0,0 +1,107 @@
+# Release 462 (16 Oct 2024)
+
+## General
+
+* Improve performance and memory consumption of the client protocol when
```

Comment:
@wendigo .. added this and the JDBC and CLI ones .. looks right?

Reply:
So it should say something like "improve Java-based Trino clients performance and memory consumption"

Reply:
ok .. then I think we can leave this entry out .. since I have a separate entry in JDBC driver and CLI .. sounds good?

Reply:
Yep. Makes sense